.cirrus.yml

```diff
@@ -47,12 +47,44 @@ centos_8_1_task:
     image: quay.io/avocado-framework/avocado-vt-ci-centos-8.1
   env:
     matrix:
-      - AVOCADO_SRC:
-      - AVOCADO_SRC: avocado-framework<70.0
+      # Older LTS release is 82.x
+      - AVOCADO_SRC: avocado-framework<83.0
+      # Latest LTS release is 92.x
+      - AVOCADO_SRC: avocado-framework<93.0
+    matrix:
+      - SETUP: setup.py develop --user
+      - SETUP: -m pip install .
+  setup_script:
+    *setup_scr
+  bootstrap_script:
+    *bootstrap_scr
+  list_script:
+    *list_scr
+  dry_run_script:
+    *dry_run_scr
+
+avocado_devel_task:
+  container:
+    image: quay.io/avocado-framework/avocado-vt-ci-fedora-34
```

Comment:
do you mind using quay.io/avocado-framework/avocado-vt-ci-fedora-35 instead?
This image is already available in quay.